### PR TITLE
Describe how to release warden

### PR DESCRIPTION
**Pull request by kwiatekus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,6 +76,30 @@ make manifests
...
+
+If you want to patch a version, cherry pick all fix commits into the release branch.
+
+If you want to create a new release, create a new branch from main for the release.
+
+For both cases, once all changes are in the release branch you need to do the following:
+
+ - create a new tag for the release using semantic versioning, i.e `v0.2.0-rc2`
+   ```bash
+   git tag -a v0.2.0-rc2 -m "v0.2.0-rc2"
```
Comment (dbadura) [inline README.md:91]: We could use `env` like `RELEASE_NAME=v0.2.0-rc3

File: README.md [hunk 2]
```diff
@@ -76,6 +76,30 @@ make manifests
...
+   git push upstream v0.2.0-rc2
+   ```
+   
+ - go to warden project on github and create release from the new tag
+
+ - checkout locally from the tag and create an archive
+   ```bash
+   tar -czf warden-0.2.0-rc2.tgz warden
```
Comment (dbadura) [inline README.md:99]: You have to enter `chart` dir.

File: README.md (hunk 1 above)
Comment (dbadura) [inline README.md:91]: Please use long version of options: `-a` and `-m`

Comment (dbadura) [inline README.md:91]: Do we need to use `--message`?

File: README.md [hunk 3]
```diff
@@ -76,6 +76,30 @@ make manifests
...
+
+For both cases, once all changes are in the release branch you need to do the following:
+
+ - create a new tag for the release using semantic versioning, i.e `v0.2.0-rc2`
+   ```bash
+   git tag -a v0.2.0-rc2 -m "v0.2.0-rc2"
+   git push upstream v0.2.0-rc2
+   ```
+   
+ - go to warden project on github and create release from the new tag
```
Comment (dbadura) [inline README.md:95]: We can create release after creating the archive. There won't be need to edit the release.

File: README.md (hunk 1 above)
Comment (dbadura) [reply README.md:91]: Yes, it's needed

File: README.md (hunk 3 above)
Comment (anoipm) [reply README.md]: description changed

File: README.md (hunk 2 above)
Comment (anoipm) [reply README.md]: added

**Review by anoipm — APPROVED** [review]

**Pull request merged**